src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozerList = new ArrayList<>();
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        Bulldozer bulldozer3 = new Bulldozer();
```

Comment:
```suggestion
        Bulldozer firstBulldozer = new Bulldozer();
        Bulldozer secondBulldozer = new Bulldozer();
        Bulldozer thirdBulldozer = new Bulldozer();
```
the same for other producers